index.d.ts

```diff
@@ -133,8 +133,10 @@ declare module '@primer/components' {
   export const CircleOcticon: React.FunctionComponent<CircleOcticonProps>
 
   export interface StyledOcticonProps extends CommonProps {
-    size?: number
+    size?: number | 'small' | 'medium' | 'large'
     icon: React.ReactNode
```

Comment:
I think this also needs to be changed, but I'm not sure to what. Type `prop-types` DefinitelyTyped package defines a `ReactComponentLike` which [they use for `PropTypes.elementType`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/68523d90957c0568592f7bea00cde8a809063fbc/types/prop-types/index.d.ts#L68).

```typescript
export type ReactComponentLike =
    | string
    | ((props: any, context?: any) => any)
    | (new (props: any, context?: any) => any);
```